### PR TITLE
feature-medical-professional-credentials

### PR DESCRIPTION
**Pull request by giovanniorigins**
views and functionality (80%)

### CODE REVIEW

File: app/Http/Controllers/Api/UtilitiesController.php [hunk 1]
```diff
@@ -14,9 +14,21 @@
 
 class UtilitiesController extends Controller
 {   
-    public function getTeamRoles()
+    public function getTeamRoles($type = null)
```
Comment (nkeena) [inline app/Http/Controllers/Api/UtilitiesController.php]: Maybe we should move this to the TeamRole class itself?

File: app/Http/admin.php [hunk 2]
```diff
@@ -80,6 +80,11 @@
     return view('admin.records.medical-releases.show', compact('release'));
 });
 
+Route::get('records/media-professionals/{id}', function ($id) use ($dispatcher) {
+    $release = $dispatcher->get('professionals/media/' . $id);
```
Comment (nkeena) [inline app/Http/admin.php]: The API will be `credentials/media`

File: app/Http/Controllers/Api/UtilitiesController.php (hunk 1 above)
Comment (giovanniorigins) [inline app/Http/Controllers/Api/UtilitiesController.php]: Which ever is the most efficient

File: app/Http/admin.php (hunk 2 above)
Comment (giovanniorigins) [inline app/Http/admin.php]: that wasn't even supposed to be there haha

**Pull request merged**